markdown/renderer.go

```diff
@@ -23,22 +23,35 @@ var (
 	codeBlockChars          = []byte("```")
 	tableHeaderColChar      = []byte{'-'}
 	tableHeaderAlignColChar = []byte{':'}
+	heading1UnderlineChar   = []byte{'='}
+	heading2UnderlineChar   = []byte{'-'}
 )
 
 // Ensure compatibility with Goldmark parser.
 var _ renderer.Renderer = &Renderer{}
 
 // Renderer allows to render markdown AST into markdown bytes in consistent format.
 // Render is reusable across Renders, it holds configuration only.
-type Renderer struct{}
+type Renderer struct {
+	underlineHeadings bool
+}
+
+func (mr *Renderer) AddOptions(...renderer.Option) {
+	// goldmark weirdness, just ignore (called with just HTML options...)
+}
 
-func (mr *Renderer) AddOptions(opts ...renderer.Option) {
-	c := renderer.Config{}
+func (mr *Renderer) AddMarkdownOptions(opts ...Option) {
 	for _, o := range opts {
-		o.SetConfig(&c)
+		o(mr)
 	}
+}
+
+type Option func(r *Renderer)
 
-	// TODO(bwplotka): Add headers optionality (https://github.com/Kunde21/markdownfmt/issues/14).
+func UnderlineHeadings(enable bool) Option {
```

Comment:
```suggestion
func WithUnderlineHeadings() Option {
```
Can we have still this? 🤗

Reply:
sure

Reply:
fixed. For whatever reason github is still not showing it, maybe caching issue